models/issue_reaction.go

```diff
@@ -160,6 +184,16 @@ func DeleteCommentReaction(doer *User, issue *Issue, comment *Comment, content s
 	})
 }
 
+// LoadUser load user of reaction
+func (r *Reaction) LoadUser() (*User, error) {
+	user, err := getUserByID(x, r.UserID)
```

Comment:
We should check if user has been loaded.

Reply:
@lunny something like:
```diff
diff --git a/models/issue_reaction.go b/models/issue_reaction.go
index 95ca76f5b..2548457b2 100644
--- a/models/issue_reaction.go
+++ b/models/issue_reaction.go
@@ -186,6 +186,9 @@ func DeleteCommentReaction(doer *User, issue *Issue, comment *Comment, content s
 
 // LoadUser load user of reaction
 func (r *Reaction) LoadUser() (*User, error) {
+       if r.User != nil {
+               return r.User, nil
+       }
        user, err := getUserByID(x, r.UserID)
        if err != nil {
                return nil, err
```
?

Reply:
done

Reply:
@lunny done